<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loadMongoDBDialogBase</class>
 <widget class="QDialog" name="loadMongoDBDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load MongoDB Points</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>579</width>
        <height>487</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Connection</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Server:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Database:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Geometry field:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0" colspan="3">
               <widget class="QTreeWidget" name="listCol">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <attribute name="headerDefaultSectionSize">
                 <number>210</number>
                </attribute>
                <attribute name="headerMinimumSectionSize">
                 <number>200</number>
                </attribute>
                <attribute name="headerStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Items</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string notr="true">Geometry</string>
                 </property>
                 <property name="toolTip">
                  <string notr="true"/>
                 </property>
                 <property name="statusTip">
                  <string notr="true"/>
                 </property>
                 <property name="whatsThis">
                  <string notr="true"/>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Count</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="load_collection">
                <property name="text">
                 <string>Load</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="createFile">
                <property name="text">
                 <string>Connect</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="load_field">
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="Close_button">
             <property name="text">
              <string>Close</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="serverName">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QComboBox" name="dbName">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QComboBox" name="geom_field">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Settings</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QListWidget" name="view_all">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="view_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>View</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="distinct_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Distinct</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="set_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QListWidget" name="view_distinct">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serverName</tabstop>
  <tabstop>dbName</tabstop>
  <tabstop>geom_field</tabstop>
  <tabstop>createFile</tabstop>
  <tabstop>load_collection</tabstop>
  <tabstop>listCol</tabstop>
  <tabstop>Close_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Close_button</sender>
   <signal>clicked()</signal>
   <receiver>loadMongoDBDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
